components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +159,556 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
     {
       "effect": "ALLOW",
       "actions": [
-        "iam:users:list"
-      ]
+        "iam:users:list",
+        "iam:users:get"
+      ],
+      "projects": [
+        "*"
+      ],
     },
     {
       "effect": "ALLOW",
       "role": "editor"
     }
+  ],
+  "projects": [
+    "admin-group-1",
+    "it-support",
   ]
 }
 ```
 
-Assuming you stored the above JSON in the file "policy.json", pass it to `curl` to create the policy:
+### Listing Policies
 
-```bash
-curl -s -H "api-token: $TOKEN" -d @policy.json https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
-```
+The output includes both *Chef-managed* and *Custom* policies.
+Policies migrated from IAM v1 are included as *Custom* policies.
+Policy names that begin with `[Legacy]` are default policies migrated from v1.
+We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
 
-### Updating a Policy
+*In the browser:* **Settings**  >> **Policies**
 
-In the IAM beta, policies must be updated from the command line.
+### Getting a Policy
 
-* You can modify the *membership* of both Chef-managed and custom policies; see [Policy Membership]({{< relref "iam-v2-api-reference.md#policy-membership" >}}) for more information.
-* You can modify the *definition* of custom policies but not Chef-managed policies.
-* You can change the policy name, members, or statements, but you cannot change the policy ID.
-* Modifying any part of a policy resets *all* of its properties.
-  Properties that aren't included in the command are reset to empty values.
-  Thus, you must supply all of a policy's properties, not just the ones you wish to update.
+In order to fetch a policy on the command-line, you must know its ID.
+This may be obtained by first fetching the list of all policies and getting the `id` field of the policy of interest.
```

Comment:
We wrote this before we'd built out the UI, so now we have some easier ways to do this 💁